### DISCUSSION TITLE
Unable to delete storage folder

### DISCUSSION THREAD

**Discussion by dilly-sixmo** [supabase/supabase#9527] [General]
I was attempting to use the Supabase API to create a subfolder within one of my storage buckets but it resulted in this issue.

I ran `.createBucket('company/test', { public: false })` and it executed successfully, but instead of creating a subfolder within the `company` bucket, it created a new bucket named `company/test`.  Now, I'm unable to delete this folder from within the admin panel:

![image](https://user-images.githubusercontent.com/112413500/195621016-bada3796-6f1d-4beb-8948-8858a2dc06aa.png)

Clicking "delete bucket" here takes me to a 404 page.

**Comment by GaryAustin1** [comment]
There really is no such thing as a sub folder in storage, it is all just part of the path.  That is just a UI concept and they create fake empty files with a special name to act as a folder.

Seems like you should generate an issue on the API letting you create a bucket with `/` in it.  The UI stops it: `Only lowercase letters, numbers, dots, and hyphens`.

You can get rid of the bucket by using sql to delete that row from the bucket table.   These buckets do not exist in s3 just in the bucket table.

**Reply by dilly-sixmo** [reply]
Thank you.

I was able to delete the bucket with an SQL query.

```
DELETE FROM storage.buckets
WHERE id = 'company/test';
```
